types/resource.go

```diff
@@ -191,17 +191,20 @@ type NodeSpec struct {
 	AllowScheduling bool                `json:"allowScheduling"`
 }
 
-type NodeState string
+type NodeConditionType string
 
 const (
-	NodeStateUp   = NodeState("up")
-	NodeStateDown = NodeState("down")
+	NodeConditionTypeReady = "Ready"
+)
+
+const (
+	NodeConditionReasonNodeDown = "NodeDown"
```

Comment:
`NodeConditionReasonManagerPodDown`

NodeDown is the result, not the reason.

If we detect the Kubernetes node down, that would be `KubernetesNodeDown`